packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts

```diff
@@ -0,0 +1,66 @@
+import { IdentityDeletionProcessStatus } from "@nmshd/transport";
+import { PeerDeletedEvent, PeerDeletionCancelledEvent, PeerStatus, PeerToBeDeletedEvent } from "../../src";
+import { establishRelationship, RuntimeServiceProvider, TestRuntimeServices, waitForEvent } from "../lib";
+
+const serviceProvider = new RuntimeServiceProvider();
+let services1: TestRuntimeServices;
+let services2: TestRuntimeServices;
+let relationshipId: string;
+
+beforeAll(async () => {
+    const runtimeServices = await serviceProvider.launch(2);
+    services1 = runtimeServices[0];
+    services2 = runtimeServices[1];
+    relationshipId = (await establishRelationship(services1.transport, services2.transport)).id;
+}, 30000);
+
+beforeEach(() => {
+    services2.eventBus.reset();
+});
+
+afterAll(async () => {
+    return await serviceProvider.stop();
+});
+
+afterEach(async () => {
+    const activeIdentityDeletionProcess = await services1.transport.identityDeletionProcesses.getActiveIdentityDeletionProcess();
+    if (!activeIdentityDeletionProcess.isSuccess) {
+        return;
+    }
+    let abortResult;
+    if (activeIdentityDeletionProcess.value.status === IdentityDeletionProcessStatus.Approved) {
+        abortResult = await services1.transport.identityDeletionProcesses.cancelIdentityDeletionProcess();
+    } else if (activeIdentityDeletionProcess.value.status === IdentityDeletionProcessStatus.WaitingForApproval) {
+        abortResult = await services1.transport.identityDeletionProcesses.rejectIdentityDeletionProcess();
+    }
+
+    if (abortResult?.isError) throw abortResult.error;
+});
+
+describe("IdentityDeletionProcess", () => {
+    test("peer should be notified about started deletion process", async function () {
+        await services1.transport.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        await services1.transport.identityDeletionProcesses.approveIdentityDeletionProcess();
```

Comment:
This line is not needed (and should actually throw an error). Initiating an IdentityDeletionProcess directly moves it to status Approved. `approveIdentityDeletionProcess` is called after an IdentityDeletionProcess was started via the Admin UI instead of by the user directly.

Reply:
@Magnus-Kuhn if this should really throw an error please look why it not does before removing it.

Reply:
the result was an error, but that doesn't stop the test

Reply:
Hmmm, typically our test utils throw..